README.md

```diff
@@ -70,7 +70,9 @@ arpack-ng is replacing arpack almost everywhere.
 
    The following instructions explain how to make the ARPACK library.
 
-4. Unlike ARPACK, ARPACK-NG is providing autotools based build system.
+4. Unlike ARPACK, ARPACK-NG is providing autotools based build system
+   and iso_c_binding support (which enables to call fortran subroutines
```

Comment:
maybe you could add a link to a page which explains what is iso_c_binding?

Reply:
Woaw !... Not so easy. I try to get one.
I know a good one in french (in idris website - where I learnt everything from scratch), but, if I remember well this one is not translated in english !... I'll try to find something.

Reply:
My understanding is that IDRIS lectures are french only. But I found:
http://fortranwiki.org/fortran/show/ISO_C_BINDING
http://fortranwiki.org/fortran/show/Generating+C+Interfaces

Do you buy one of these or both ?!

Reply:
Yeah, clearly!